openmdao/docs/theory_manual/total_derivs/matrix_figs/jac_structure.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from pyxdsm.matrix_eqn import MatrixEquation
```

Comment:
pyxdsm dependency?

Reply:
Actually, never mind. I assume these are just reference for generating the figures.

Reply:
yeah. You would need pyxdsm to regenerate the figures. but its not a dependency